examples/notebooks/beam-ml/run_inference_pytorch_tensorflow_sklearn.ipynb

```diff
@@ -41,25 +36,23 @@
         "# KIND, either express or implied. See the License for the\n",
         "# specific language governing permissions and limitations\n",
         "# under the License"
-      ]
-    },
-    {
-      "cell_type": "markdown",
+      ],
       "metadata": {
+        "cellView": "form",
         "id": "faayYQYrQzY3"
-      },
-      "source": [
-        "## Use RunInference in Apache Beam"
-      ]
+            },
+      "execution_count": null,
+      "outputs": []
     },
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "JjAt1GesQ9sg"
       },
       "source": [
-        "Starting with Apache Beam 2.40.0, you can use Apache Beam with the RunInference API to use machine learning (ML) models for local and remote inference with batch and streaming pipelines.\n",
-        "The RunInference API leverages Apache Beam concepts, such as the BatchElements transform and the Shared class, to support models in your pipelines that create transforms optimized for machine learning inferences.\n",
+        "# Use RunInference in Apache Beam\n",
+        "You can use Apache Beam versions 2.40.0 and later with the [RunInference API](https://beam.apache.org/releases/pydoc/current/apache_beam.ml.inference.base.html#apache_beam.ml.inference.base.RunInference) to use machine learning (ML) models for local and remote inference with batch and streaming pipelines.\n",
```

Comment:
"You can use Apache Beam versions 2.40.0 and later with the RunInference API to use machine learning (ML) models for local and remote inference with batch and streaming pipelines." - this reads a little awkwardly to me, specifically "you can use X to use Y"

Maybe "You can use Apache Beam versions 2.40.0 and later with the RunInference API for local and remote inference with batch and streaming pipelines" instead? The "to use machine learning (ML) models" bit is probably implied

Reply:
Updated, thanks!